inferelator_ng/tfa.py

```diff
@@ -31,12 +31,14 @@ class TFA:
     def __init__(self, prior, exp_mat, exp_mat_halftau):
         self.prior = prior
         self.exp_mat = exp_mat
-        self.exp_mat = exp_mat_halftau
+        self.exp_mat_halftau = exp_mat_halftau
 
     def tfa(self, noself = True, dup_self = True):
         tfwt = self.prior.abs().sum(axis = 0) > 0
         tfwt = tfwt[tfwt].index.tolist()
 
+        dTFs = []
```

Comment:
We usually use default options. But it might be a good idea to leave it there as we might find a way to deal with other scenarios later:)